src/exceptions/x11vnc-openssl-exception.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="x11vnc-openssl-exception"
```

Comment:
I think exceptions need the tag to be "exception" rather than "license"...

Reply:
I'm also not sure whether `isOsiApproved` can (or should) be an attribute on an `exception` tag...?

Reply:
good catch, thanks!